src/flag_gems/ops/fill.py

```diff
@@ -0,0 +1,60 @@
+import logging
+
+import torch
+import triton
+import triton.language as tl
+
+from ..utils import libentry
+
+
+@libentry()
+@triton.jit
```

Comment:
recommend to register value_scalar as do_not_specialize

Reply:
Fixed